danilobassi8/prode
# Integrate previous bets with backend

This PR integrates the previous bet tree with the backend. I wont style this components until we're sure that its functionality works at 100%.

HOW TO TEST:
- You'll need to have the backend branch updated.
- First, go to the `/login` url, and login. (wont work if no logged in)
- Go to `/pre-mundial` url and test it. First time will be empty, but once saved, should be populated correctly if you press F5.

If this works and get merged, I'll start with the final styles. 🙏🏼
